app/models/calculation_result.rb

```diff
@@ -1,46 +1,27 @@
 class CalculationResult
```

Comment:
query - should we change the name of this class to be `CalculateResultForView` or is that a bit much?

Reply:
Good spot, the name is not ideal especially now we have extracted out the logic that pulls out the data. But we're not sure what the name should be so we will revisit that later on.